recipes/import-linter/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "import-linter" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/import-linter-{{ version }}.tar.gz
+  sha256: 8f682c3b10087fdba062f2d62b118fdce7fc01cc848df863e85c1a97e1194cca
+
+build:
+  entry_points:
+    - lint-imports = importlinter.cli:lint_imports_command
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7,<3.11
+    - pip
+  run:
+    - python >=3.7,<3.11
+    - click >=6
+    - grimp >=2.0
```

Comment:
This will cause a race condition and you'll have to restart the build in the feedstock.